scripts/ci/test_reborn_pr_test_plan.py

```
@@ -849,6 +874,219 @@ def test_e2e_paths_are_owned_by_dedicated_workflows(self) -> None:
+                "id": "wasm",
+                "name": "ironclaw_wasm",
+                "manifest_path": str(ROOT / "crates/ironclaw_wasm/Cargo.toml"),
+            }
+        )
+        wasm_metadata["resolve"]["nodes"].append({"id": "wasm", "deps": []})
+
+        plan = planner.build_plan(
+            event="pull_request",
+            changed_paths=["crates/ironclaw_wasm/witless/notes.txt"],
+            metadata=wasm_metadata,
+            canonical_packages=self.canonical + ["ironclaw_wasm"],
```

Ruff (0.16.1)

[warning] Consider `[*self.canonical, "ironclaw_wasm"]` instead of concatenation

Replace with `[*self.canonical, "ironclaw_wasm"]`

(RUF005)